scripts/Makefile

```diff
@@ -50,24 +50,25 @@ all: $(GENERATED_FILES)
 			order by fiss_density_ids"
 
 ../data/fiss_density_watersheds.geojson: ../data/fiss_density_pts.geojson
-	#$(PSQL_CMD) -f sql/05_fiss_density_watersheds.sql  # create watersheds
-	# dump to geojson and overlay with elevation raster
+	$(PSQL_CMD) -f sql/05_fiss_density_watersheds.sql  # create watersheds
+	# dump to temp geojson file
 	$(PSQL_CMD) -Xt <<< "select \
 		st_asgeojson(t.*) \
 		from \
 		( select \
 			fiss_density_watersheds_id, \
-			fiss_density_ids, \
+			array_to_string(fiss_density_ids, ';') as fiss_density_ids, \
```

Comment:
is this the latest version?

Reply:
No, latest is in 2nd commit. https://github.com/NewGraphEnvironment/fissr_explore/pull/10/commits/664d110f6b32e936a2eecbeba837d286b6829ec4